<?xml version="1.0"?>
<launch>
<arg name="uavID" default="uav0"/>
   <group ns="$(arg uavID)" >
	<node pkg="px4_command" type="px4_multidrone_pos_controller" name="px4_multidrone_pos_controller" output="screen" args="$(eval arg('uavID')[-1:])">
	<rosparam command="load" file="$(find px4_command)/config/Parameter_for_control_gazebo.yaml" />
	</node>
   </group>
</launch>
